website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -51,6 +51,26 @@ The volume staging directory for new CSI plugin tasks will now be
 mounted to the task's `NOMAD_TASK_DIR` instead of the
 `csi_plugin.mount_config`.
 
+#### Server Raft Database
+
+The server raft database in `raft.db` will be automatically migrated to a new
+underlying implementation provided by `go.etcd.io/bbolt`. Downgrading to a previous
+version of the server after upgrading it to Nomad 1.3 is not supported.
```

Comment:
Maybe we should be a little more prescriptive to help folks out with our No Going Back upgrades:

>  Like any Nomad upgrade it is recommended that you take a snapshot of your database prior to upgrading to use in case you need to downgrade.

Heavily inspired by https://www.consul.io/docs/upgrading/upgrade-specific#raft-storage-changes but removed the `if you expect that you will need to downgrade.` because who ever _expects_ they'll need to downgrade? :grimacing:

Reply:
:+1: